Sources/SwiftSyntax/SwiftSyntaxCompatibility.swift

```diff
@@ -85,6 +85,60 @@ public extension EnumDeclSyntax {
   }
 }
 
+public extension FunctionTypeSyntax {
+  @available(*, deprecated, renamed: "unexpectedBetweenLeftParenAndParameters")
+  var unexpectedBetweenLeftParenAndArguments: UnexpectedNodesSyntax? { unexpectedBetweenLeftParenAndParameters }
+
+  @available(*, deprecated, renamed: "parameters")
+  var arguments: TupleTypeElementListSyntax { parameters }
```

Comment:
Ah, I totally forgot that these properties also need to be settable. Would you mind also changing all the other properties in this compatibility layer to be settable?

Reply:
Fixed